test/builders/RequestBuilder.scala

```diff
@@ -50,7 +50,7 @@ object RequestBuilder {
       .withHeaders(headers.toArray: _*)
 
   def buildFakeRequestWithAuth(method: String) =
-    FakeRequest(method = method, path = "")
+    FakeRequest(method = method, path = "/")
```

Comment:
how come this was changed?

Reply:
Not entirely sure of this @brendanstraughan but this is coming from  "Is this page not working properly?"

The tai application is now calling the _contact-frontend_ service asking us to present a URL PATH “/”   whereas previously it was inbuilt so the PATH was not needed